testing/endtoend/evaluators/beaconapi_evaluators/beacon_api.go

```diff
@@ -242,30 +508,166 @@ var beaconPathsAndObjects = map[string]metadata{
 		lighthouseResps: map[string]interface{}{
 			"json": &node.GetPeersResponse{},
 		},
+		customEvaluation: func(p interface{}, l interface{}) error {
+			pResp, ok := p.(*node.GetPeersResponse)
+			if !ok {
+				return errJsonCast
+			}
+			lResp, ok := l.(*node.GetPeersResponse)
+			if !ok {
+				return errJsonCast
+			}
+			if pResp.Data == nil {
+				return errEmptyPrysmData
+			}
+			if lResp.Data == nil {
+				return errEmptyLighthouseData
+			}
+			return nil
+		},
+	},
+	"/node/peer_count": {
+		basepath: v1PathTemplate,
+		params: func(_ string, _ primitives.Epoch) []string {
+			return []string{}
+		},
+		prysmResps: map[string]interface{}{
+			"json": &node.GetPeerCountResponse{},
+		},
+		lighthouseResps: map[string]interface{}{
+			"json": &node.GetPeerCountResponse{},
+		},
+		customEvaluation: func(p interface{}, l interface{}) error {
+			pResp, ok := p.(*node.GetPeerCountResponse)
+			if !ok {
+				return errJsonCast
+			}
+			lResp, ok := l.(*node.GetPeerCountResponse)
+			if !ok {
+				return errJsonCast
+			}
+			if pResp.Data == nil {
+				return errEmptyPrysmData
+			}
+			if lResp.Data == nil {
+				return errEmptyLighthouseData
+			}
+			return nil
+		},
+	},
+	"/node/version": {
+		basepath: v1PathTemplate,
+		params: func(_ string, _ primitives.Epoch) []string {
+			return []string{}
+		},
+		prysmResps: map[string]interface{}{
+			"json": &node.GetVersionResponse{},
```

Comment:
since this is a bunch of copy and paste maybe we can make a copy of this object for lighthouse responses? and not require it anymore if it's the same?

Reply:
I think explicitly creating two structs is safer. This is because if you want to specify the object only once and then copy it, creating a pointer with `&` will lead to wrong behavior because a copy will copy the pointer, and therefore both responses will always match. We would have to be very careful to not use `&`, which is risky.